src/Navigation/Education/index.js

```diff
@@ -1,9 +1,35 @@
-import React from "react";
-function Education(props, ref) {
+import React, { useEffect } from "react";
+import "./index.css";
+import EducationContent from "./EducationContent";
+import EducationHeader from "./EducationHeader";
+const Education = ({ educationData }, ref) => {
+  const handleScroll = () => {
+    if (document.documentElement.scrollTop >= ref.current.offsetTop) {
```

Comment:
With this condition, animation starts when Education section is at the top of page, we should start animation when the section is at middle.
Condition should be something like this: `ScrollTop  > (EducationSection.OffsetTop - EducationSection.Height/2)`